src/UniqueCharacters.java

```diff
@@ -1,10 +1,19 @@
+import java.util.Scanner;
+
 public class UniqueCharacters {
     public static void main(String[] args) {
-        System.out.println(uniqueCharacters("caio"));
+        Scanner input = new Scanner(System.in);
+        System.out.println("Welcome to UniqueCharacters!!\nEnter a string!!");
```

Comment:
put the /welcome message at the first line of the main method

Reply:
Done.